frontend/integration-tests/protractor.conf.ts

```diff
@@ -109,8 +108,7 @@ export const config: Config = {
       'tests/modal-annotations.scenario.ts',
       'tests/source-to-image.scenario.ts',
       'tests/deploy-image.scenario.ts',
-      'tests/operator-hub/operator-hub.scenario.ts',
-      'tests/catalog.scenario.ts'],
```

Comment:
Do we want to keep developer-catalog.scenario in all?

Reply:
No, it's now part of the service-catalog scenario, which is included in 'all'

Reply:
It needs to be in the service-catalog folder to match the wildcard `service-catalog/**` wildcard above. Otherwise you’d need to list developer-catalog.ts explicitly here.

Reply:
yes, right, sorry -added `developer-catalog.ts` back to 'all'